examples/guestbook/README.md

```diff
@@ -199,111 +302,73 @@ If you want to see the logs for a given container, you can run:
 $ docker logs <container_id>
 ```
 
-### Step Two: Fire up the redis master service
+### Step Two: Start up the redis slave
 
-A Kubernetes [service](../../docs/user-guide/services.md) is a named load balancer that proxies traffic to one or more containers. This is done using the [labels](../../docs/user-guide/labels.md) metadata that we defined in the `redis-master` pod above.  As mentioned, we have only one redis master, but we nevertheless want to create a service for it.  Why?  Because it gives us a deterministic way to route to the single master using an elastic IP.
+Now that the redis master is running, we can start up its 'read slaves'.
 
-Services find the pods to load balance based on the pods' labels.
-The pod that you created in [Step One](#step-one-start-up-the-redis-master) has the label `name=redis-master`.
-The selector field of the service description determines which pods will receive the traffic sent to the service, and the `port` and `targetPort` information defines what port the service proxy will run at.
+We'll define these as replicated pods as well, though this time— unlike for the redis master— we'll define the number of replicas to be 2.
+In Kubernetes, a replication controller is responsible for managing multiple instances of a replicated pod. The replication controller will automatically launch new pods if the number of replicas falls below the specified number.
+(This particular replicated pod is a great one to test this with -- you can try killing the docker processes for your pods directly, then watch them come back online on a new node shortly thereafter.)
 
-The file `examples/guestbook/redis-master-service.yaml` defines the redis master service:
+Just like the master, we want to have a service to proxy connections to the redis slaves. In this case, in addition to discovery, the slave service will provide transparent load balancing to web app clients.
 
-<!-- BEGIN MUNGE: EXAMPLE redis-master-service.yaml -->
+This time we put the service and RC into one [file](../../docs/user-guide/managing-deployments.md#organizing-resource-configurations).
```

Comment:
Use separate files in redis-maser section is more easy to talk about pod and service at the beginning.